R/install-local.r

```diff
@@ -46,3 +46,13 @@ remote_metadata.local_remote <- function(x, bundle = NULL, source = NULL) {
     RemoteSha = if (uses_git(x$path)) git_sha1(path = x$path)
   )
 }
+
+#' @export
+remote_metadata.package <- function(x, bundle = NULL, source = NULL) {
```

Comment:
If we change this to the following it will give us consistent results with the legacy behavior if the package is not using git or has an unclean working directory.

```r
compact(list(
    RemoteType = "local",
    RemoteUrl = x$path,
    RemoteSubdir = x$subdir,
    RemoteSha = if (git_committed(x$path)) git_sha1(path = x$path)
  ))
```